src/open_lib.cpp

```diff
@@ -197,24 +200,39 @@ static const char *get_path_from_win32_projlib(const char *name, std::string& ou
 #endif
 
 /************************************************************************/
-/*                          pj_open_lib_ex()                            */
+/*                      pj_open_lib_internal()                          */
 /************************************************************************/
 
-static PAFile
-pj_open_lib_ex(projCtx ctx, const char *name, const char *mode,
-               char* out_full_filename, size_t out_full_filename_size) {
-    try {
-        std::string fname;
-        const char *sysname = nullptr;
-        PAFile fid = nullptr;
 #ifdef WIN32
-        static const char dir_chars[] = "/\\";
-        const char dirSeparator = ';';
+static const char dir_chars[] = "/\\";
```

Comment:
C++17 has std::experimental::filesystem::path::preferred_separator. it might be worthwhile to prepare for using that.

Reply:
defering on that, especially since it is only experimental on C++17... :-)